packages/flame/lib/src/effects2/simple_effect_controller.dart

```diff
@@ -0,0 +1,53 @@
+
+import 'effect_controller.dart';
+import 'standard_effect_controller.dart';
+
+/// Simplest possible [EffectController], which supports an effect progressing
+/// linearly over [duration] seconds.
+///
+/// The [duration] can be 0, in which case the effect will jump from 0 to 1
+/// instantaneously.
+///
+/// The [delay] parameter allows to delay the start of the effect by the
+/// specified number of seconds.
+///
+/// See also: [StandardEffectController]
+class SimpleEffectController extends EffectController {
```

Comment:
What about LinerEffectController ?

Reply:
The main idea behind this controller is that it is as simple as possible (while still being useful). So it is certainly linear, but only because linear curve is the simplest. From this perspective, I think the current name is better because it conveys the limitation of this class: that we don't want to make it more complicated in the future.